tests/Unit/Adapter/Preferences/PreferencesConfigurationTest.php

```diff
@@ -29,10 +29,11 @@
 namespace Tests\Unit\Adapter\Preferences;
 
 use Cookie;
-use PHPUnit\Framework\MockObject\MockObject;
 use PHPUnit\Framework\TestCase;
 use PrestaShop\PrestaShop\Adapter\Configuration;
 use PrestaShop\PrestaShop\Adapter\Preferences\PreferencesConfiguration;
+use PrestaShop\PrestaShop\Adapter\Shop\Context as ShopContext;
```

Comment:
Shouldn't you use the abstract test class for this one? and follow the way other multistore forms have been migrated recently?